language/libra-vm/src/libra_vm.rs

```diff
@@ -282,6 +227,23 @@ impl LibraVM {
         Ok(())
     }
 
+    fn check_change_set(&self, change_set: &ChangeSet, state_view: &dyn StateView) -> VMResult<()> {
+        // TODO: Replace this logic with actual checks.
```

Comment:
Pre-existing issue, but: better TODO here? Say what additional checks we want/why these ones aren't good enough?